deployment/ansible/roles/nginx-certbot-deploy/tasks/configure_domain.yml

```diff
@@ -41,6 +41,21 @@
   when: not letsencrypt_certificate.stat.exists
   notify: Reload nginx
 
+- name: Set up basic auth
+  when: domain.auth_basic_user is defined and domain.auth_basic_user != ""
+  block:
+    - name: Install passlib
+      package:
+        name: python-passlib
```

Comment:
```
FAILED! => {"changed": false, "msg": "No package matching 'python-passlib' is available"}
```

Ini kayaknya mesti `python3-passlib` deh, soalnya ansible ini asumsinya hostnya pakai Python 3.

Reply:
Astaga python ini wkwkwk
Okay noted

Reply:
Resolved in 911efe4096de7f38c5ab11b554f76ab81aca4f06